ciao-cli/main.go

```diff
@@ -1031,7 +1038,12 @@ func main() {
 		if *imageName == "" {
 			fatalf("Missing required -image-name parameter")
 		}
-		createTenantImage(*identityUser, *identityPassword, id, *imageName)
+		if *imageVisibility != "private" && *imageVisibility != "public" {
```

Comment:
"github.com/rackspace/gophercloud/openstack/imageservice/v2/images"

already defines an ImageVisibility type,
Shouldn't it be better to include it and add the functions Set(value string) error and String() string there ?